### ISSUE TITLE
can't install it

### ISSUE THREAD

**Issue by RickMeng** [dragonwocky/notion-enhancer#31]
FILL OUT THIS TEMPLATE, REMOVING/REPLACING EVERYTHING HERE WITH YOUR OWN INFORMATION.
BE AS CLEAR AND CONCISE AS POSSIBLE.

**describe the bug**
what is the bug - what happens?
![image](https://user-images.githubusercontent.com/34000539/88586613-8b37c000-d04c-11ea-9a91-cde85e882707.png)

**expected behaviour**
why is this a bug - what should be happening?

**screenshots**
if applicable, add screenshots of your problem.

**platform**
what OS are you using? e.g. windows 7, windows 10, WSL, ubuntu, arch linux, macOS

**Comment by dragonwocky** [comment]
...I'm guessing you're on windows? Please edit the issue to fill out the template properly.

Which version of Python are you running?

Try adding `from subprocess import PIPE` to the top of `customiser.py` (on its own line), and then replacing line 42 with:
```py
        filepath = subprocess.check_output(['echo', '%localappdata%'], shell=True).stdout \
```

**Comment by dragonwocky** [comment]
fix released in v0.8.0
